Library/Homebrew/extend/os/linux/extend/pathname.rb

```diff
@@ -0,0 +1,14 @@
+class Pathname
+  # @private
+  def elf?
+    # See: https://en.wikipedia.org/wiki/Executable_and_Linkable_Format#File_header
+    read(4) == "\x7fELF"
+  end
+
+  # @private
+  def dynamic_elf?
+    return popen_read("readelf", "-l", to_path).include?(" DYNAMIC ") if which "readelf"
```

Comment:
An `if`/`elsif` in block form would make this easier to follow, I think.

Reply:
```
C: Use a guard clause instead of wrapping the code inside a conditional expression.
C: Redundant return detected.
```
@maxim-belkin If you go back to the `if/else` code that you had before, and remove the `return` statements, it should clear up both of these style errors.